terps/tads/glk/osportable.cc

```diff
@@ -48,11 +48,14 @@
 #include "osextra.h"
 #include "charmap.h"
 #include <time.h>
+#ifndef _WIN32
```

Comment:
This file comes from upstream frobtads, which is maintained.  Ideally this code would be upstreamed so that we don't have to patch it here in future releases. It's at least worth checking with Nikos to see if he's interested in adding this Windows support.  If not, no problem, we can continue to keep it here.  But as much as possible I want to avoid deviating from upstream interpreters.

Reply:
Correction: this is from https://github.com/tads-intfic/tads-runner, not FrobTADS, though they share lots of code.

So it's worth seeing if _they_ are interested in these changes.

Reply:
Of course!